core/src/main/java/hudson/scm/SCM.java

```diff
@@ -739,7 +741,7 @@ protected final boolean createEmptyChangeLog(File changelogFile, BuildListener l
      * @since 1.568
      */
     protected final void createEmptyChangeLog(@NonNull File changelogFile, @NonNull TaskListener listener, @NonNull String rootTag) throws IOException {
-        try (FileWriter w = new FileWriter(changelogFile)) {
+        try (Writer w = Files.newBufferedWriter(Util.fileToPath(changelogFile), Charset.defaultCharset())) {
```

Comment:
Hmm, I guess we do need to use platform charset for compatibility here. (In most cases the file is _not_ XML despite appearances.)